tests/pos/byname-inline.scala

```diff
@@ -0,0 +1,11 @@
+final class ByName1(val x: Int) extends AnyVal:
+  inline def isEmpty: Boolean = false
```

Comment:
This only seems to be a problem when `isEmpty` returns a literal, or calls a function returning a literal type.